### DISCUSSION TITLE
为什么 立刻安装依赖? n 但是又自动装了nonebot-plugin-gocqhttp（

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [CMHopeSunshine/LittlePaimon#399] [Q&A]
![image](https://user-images.githubusercontent.com/51957264/222507413-257994b1-9c3a-4216-a13b-448842be349d.png)

**Comment by CMHopeSunshine** [comment]
你不是选了gocq插件版吗，插件版就是需要依赖

**Reply by shoucandanghehe** [reply]
意思是 `立刻安装依赖` 选了 n
应该把 `nonebot-plugin-gocqhttp` 写到 `pyproject.toml` 的依赖声明里 而不是就地用 pip 装上
或者放到后面的提醒里让用户自己到虚拟环境里装🤔
